contracts/ETHDenverStaking.sol

```diff
@@ -8,80 +8,82 @@ import "../node_modules/zeppelin-solidity/contracts/ECRecovery.sol";
 
 
 contract ETHDenverStaking is Ownable, Pausable {
-  
+
     using SafeMath for uint256;
     using ECRecovery for bytes32;
 
-    event UserStake(address userAddress, address stakedBy, uint amountStaked);
-    event UserRecoupStake(address userAddress, address stakedBy, uint amountStaked);
```

Comment:
Would prefer to keep the events with the previous arguments names to avoid changing the listener already in place. Maybe a comment is enough?